lib/libc/mips/string/memcpy_c.S

```diff
@@ -75,6 +75,15 @@ memcpy_c:
 	andi     $12, $v0, CAP_ALIGN_MASK # t4 = dst % CAP_SIZE
           CToPtr   $v0, $c0, $c3       # Return ptr in integer return
 	andi     $13, $v1, CAP_ALIGN_MASK # t5 = src % CAP_SIZE
+
+	CGetPerm $a5, $c3
```

Comment:
This is quite ugly and was intended as a quick hack.  It's probably okay for now, but I don't like it.